R/makephyloseqFromTreeSummarizedExperiment.R

```diff
@@ -101,6 +101,7 @@ setMethod("makePhyloseqFromTreeSE",
             # Gets the taxonomic data from rowData, and converts it to tax_table
             tax_table <- as.matrix(rowData(x)[,taxonomyRanks(x)])
```

Comment:
The problem seems to be that only genus level is available. By default drop parameter is TRUE, which means that vector is returned and rownames are lost

-->

as.matrix( rowData(x)[,taxonomyRanks(x), drop = FALSE] )

Reply:
Yes that's right and now it works.Thanks! 
Thought it has to do with the tax_table row names